temporalio/test/test.rb

```diff
@@ -139,15 +142,32 @@ class TestEnvironment
     attr_reader :server
 
     def initialize
-      @server = Temporalio::Testing::WorkflowEnvironment.start_local(
-        logger: Logger.new($stdout),
-        dev_server_extra_args: [
-          # Allow continue as new to be immediate
-          '--dynamic-config-value', 'history.workflowIdReuseMinimalInterval="0s"'
-        ]
-      )
-      Minitest.after_run do
-        @server.shutdown
+      # Start workflow env for an existing server if env vars present
+      if ENV['TEMPORAL_TEST_CLIENT_TARGET_HOST']
+        client = Temporalio::Client.connect(
+          ENV['TEMPORAL_TEST_CLIENT_TARGET_HOST'],
+          ENV['TEMPORAL_TEST_CLIENT_TARGET_NAMESPACE'] || 'default',
+          data_converter: Temporalio::Converters::DataConverter.default,
+          interceptors: [],
+          logger: Logger.new($stdout, level: Logger::WARN),
+          default_workflow_query_reject_condition: nil,
+          runtime: Temporalio::Runtime.default
```

Comment:
```suggestion
          logger: Logger.new($stdout)
```

4 of these are already the default, and if we want to adjust log level, we should do it more generally instead of just for this env var (though I think default level is fine)